ford/__init__.py

```diff
@@ -355,15 +356,30 @@ def main(proj_data,proj_docs,md):
         project.make_links('..')
     else:
         project.make_links(proj_data['project_url'])
+
+    # Define core macros:
+    ford.utils.register_macro('url = {0}'.format(proj_data['project_url']))
+    ford.utils.register_macro( 'media = {0}'.format(
+                                    os.path.join(proj_data['project_url'],
+                                                 'media') ) )
+    ford.utils.register_macro( 'page = {0}'.format(
+                                    os.path.join(proj_data['project_url'],
+                                                 'page') ) )
+    # Register the user defined aliases:
+    for alias in proj_data['alias']:
+        val, key = ford.utils.register_macro(alias)
+        if key == None:
+            raise RuntimeError('Error, no alias name provided for {0}'.format(val))
```

Comment:
Might be better to just move this into `register_macro`?

Reply:
Sure, please make changes as you see fit.